pkg/datarepair/checker/checker_test.go

```diff
@@ -220,3 +236,19 @@ func BenchmarkIdentifyInjuredSegments(b *testing.B) {
 		}
 	}
 }
+
+func getDBPath() string {
+	return fmt.Sprintf("file:memdb%d?mode=memory&cache=shared", rand.Int63())
+}
+
+func getServerAndDB(path string) (s *irreparable.Server, db *dbx.DB, err error) {
```

Comment:
`func newTestIrreparableDB() (db *irreparable.DB, error)` is sufficient.

Reply:
made mods as suggested